nipype/interfaces/afni/utils.py

```diff
@@ -617,11 +617,15 @@ class CatMatvec(AFNICommand):
 
     def _format_arg(self, name, spec, value):
         if name == 'in_file':
-            return spec.argstr % (' '.join([i[0] + ' -' + i[1]
-                                            for i in value]))
+            xfm_args=''
+            for v in value:
+                if len(v[1])>0:
+                    xfm_args += ' ' +  v[0] + ' -' + v[1] + ' '
+                else:
+                    xfm_args += ' ' + v[0] + ' '
+            return spec.argstr % (xfm_args)
         return super(CatMatvec, self)._format_arg(name, spec, value)
 
-
```

Comment:
Can you re-add this? Python style guidelines suggest 2 lines between classes.

Reply:
Done